flexget/plugins/clients/rtorrent.py

```diff
@@ -387,6 +425,29 @@ def _build_options(self, config, entry, entry_first=True):
                 elif entry_value:
                     options[opt_key] = entry.render(entry_value)
 
+        # Add custom fields on to options
+        entry_custom_fields = entry.get('custom_fields')
+        config_custom_fields = config.get('custom_fields')
+
+        if entry_custom_fields is None:
+            entry_custom_fields = {}
+
+        if config_custom_fields is None:
+            config_custom_fields = {}
```

Comment:
You can use a default with get to simplify this when the key doesn't exist.
```suggestion
        entry_custom_fields = entry.get('custom_fields', {})
        config_custom_fields = config.get('custom_fields', {})
```